gitops/ckcp/base/pvc.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: kcp
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
```

Comment:
Why do we need a full Gi? If we need to store data related to certs, I assume that a few MB would be enough.

Reply:
That's just standard. I'm happy to change it 👍🏼.